internal/controllers/weatherController.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 type WeatherController struct {
-	weatherService *services.WeatherService
+	weatherService services.WeatherServer
```

Comment:
щодо того щоб оголошувати інтерфейси там де вони використовуються, додаю тут приклад

```suggestion
	type(
        WeatherController struct{
            weatherService weatherService
        }
        weatherService interface{
            GetWeather(ctx context.Context, city string) (Weather, error)
        }
    )
```

Reply:
Resolved